### ISSUE TITLE
[NEXT STEPS]

### ISSUE THREAD

**Issue by alessandro-maccario** [alessandro-maccario/DeepLearning-FoodRecognition#3]
- [x] predict phase on the same notebook as the training.
- [x] Create the last notebook, save the best model, load the best model in the new notebook, use it for predict on the new class.
- [x] increase dataset size to 500/1000 images per classes
- [x] Blurry images: detect blurry images, apply filter to correct them and then apply algorithm. You should copy the original directory already resized, apply this transformation and then check the new results from the algorithms compare to the previous ones, especially for custom architectures. Links: 
>  https://pyimagesearch.com/2015/09/07/blur-detection-with-opencv/
>  https://github.com/soorajsknair93/ComputerVision-Fun-Projects/blob/main/Blur_detection/blur_detection_fft.py
>  https://soorajsknair1993.wixsite.com/website/post/blur-detection-using-opencv-laplacian-and-fft-based-approach
>  https://stackoverflow.com/questions/58231849/how-to-remove-blurriness-from-an-image-using-opencv-python-c
>  https://en.wikipedia.org/wiki/Kernel_(image_processing)
>  https://datascience.stackexchange.com/questions/81778/cnn-how-do-i-handle-blurred-images-in-the-dataset

**Comment by alessandro-maccario** [comment]
Try out the kNN algorithm for Image compression on the training dataset.
Remember to scale/normalize the data using sklearn and save the model to scale also the input data (first notebook) in the testing phase. 

Follow:
- https://medium.com/analytics-vidhya/image-compression-using-k-means-clustering-and-principal-component-analysis-in-python-893fd37e8e15
- https://github.com/ajaleel-ds/KNN-Image-compression/blob/main/KNN%20image%20compression%20_%20Executed.ipynb
- https://www.google.com/search?q=knn+for+image+compression+python&oq=knn+for+image+compression+python&aqs=chrome..69i57j33i160j33i22i29i30.13158j0j7&sourceid=chrome&ie=UTF-8
